studio/components/interfaces/TableGridEditor/SidePanelEditor/SidePanelEditor.tsx

```diff
@@ -53,13 +56,20 @@ const SidePanelEditor: FC<Props> = ({
 
   const tables = meta.tables.list()
 
+  const { project } = useProjectContext()
+  const { mutateAsync: createTableRow } = useTableRowCreateMutation()
+  const { mutateAsync: updateTableRow } = useTableRowUpdateMutation()
+
   const saveRow = async (
     payload: any,
     isNewRecord: boolean,
     configuration: { identifiers: any; rowIdx: number },
     onComplete: Function
   ) => {
-    if (selectedTable === undefined) return
+    if (!project || selectedTable === undefined) {
+      // TODO(alaister): should we have an error state here?
```

Comment:
maybe just chuck in a console.error
but i suppose it'll never run into this block though? since there should be a project and selectedTable when the side panel editor is rendered